Code/GraphMol/MolOps.h

```diff
@@ -249,6 +249,7 @@ struct RDKIT_GRAPHMOL_EXPORT RemoveHsParameters {
   bool removeNonimplicit = true; /**< DEPRECATED equivalent of implicitOnly */
   bool updateExplicitCount =
       false; /**< DEPRECATED equivalent of updateExplicitCount */
+  bool removeHydrides = true; /**<Removing Hydrides>**/
```

Comment:
please keep the formatting of these comments consistent:
```suggestion
  bool removeHydrides = true; /**< Removing Hydrides */
```